### PR TITLE
Done Devise's Card

### PR DESCRIPTION
**Pull request by gussiciliano**
### Sumary
- Add devise to project
- Custom sing up's form with first and last name (required)
- Create a new user

### Trello Card
https://trello.com/c/Lq4HfZ0T/9-instalar-configurar-devise-y-crear-user

### CODE REVIEW

File: app/controllers/users/confirmations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class Users::ConfirmationsController < Devise::ConfirmationsController
```
Comment (Dinuuu) [inline app/controllers/users/confirmations_controller.rb]: You don't need to generate the file if you are not going to override anything

File: app/controllers/users/omniauth_callbacks_controller.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
```
Comment (Dinuuu) [inline app/controllers/users/omniauth_callbacks_controller.rb]: idem

File: app/controllers/users/passwords_controller.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+class Users::PasswordsController < Devise::PasswordsController
```
Comment (Dinuuu) [inline app/controllers/users/passwords_controller.rb]: idem

File: app/controllers/users/sessions_controller.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
+class Users::SessionsController < Devise::SessionsController
```
Comment (Dinuuu) [inline app/controllers/users/sessions_controller.rb]: idem

File: app/controllers/users/unlocks_controller.rb [hunk 5]
```diff
@@ -0,0 +1,28 @@
+class Users::UnlocksController < Devise::UnlocksController
```
Comment (Dinuuu) [inline app/controllers/users/unlocks_controller.rb]: idem

File: app/views/users/confirmations/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```
Comment (Dinuuu) [inline app/views/users/confirmations/new.html.erb]: Same for htmls, if you are not going to change anything don't create them

File: app/controllers/users/confirmations_controller.rb (hunk 1 above)
Comment (gussiciliano) [inline app/controllers/users/confirmations_controller.rb]: Oks

File: app/controllers/users/omniauth_callbacks_controller.rb (hunk 2 above)
Comment (gussiciliano) [inline app/controllers/users/omniauth_callbacks_controller.rb]: Oks

File: app/controllers/users/passwords_controller.rb (hunk 3 above)
Comment (gussiciliano) [inline app/controllers/users/passwords_controller.rb]: Oks

File: app/controllers/users/sessions_controller.rb (hunk 4 above)
Comment (gussiciliano) [inline app/controllers/users/sessions_controller.rb]: Oks

File: app/controllers/users/unlocks_controller.rb (hunk 5 above)
Comment (gussiciliano) [inline app/controllers/users/unlocks_controller.rb]: Oks

File: app/views/users/confirmations/new.html.erb (hunk 6 above)
Comment (gussiciliano) [inline app/views/users/confirmations/new.html.erb]: Oks

**Pull request merged**